README.md

```diff
@@ -1,92 +1,111 @@
-# SAS Viya 4 Infrastructure as Code (IaC) for Microsoft Azure
+# SAS Viya Infrastructure as Code (IaC) for Microsoft Azure
 
 ## Overview
 
-This project contains Terraform scripts to provision Microsoft Azure Cloud infrastructure resources required to deploy SAS Viya 4 products. Here is a list of resources this project can create -
+This project contains Terraform scripts to provision the Microsoft Azure Cloud infrastructure resources that are required to deploy SAS Viya product offerings. Here is a list of resources that this project can create:
 
-  >- Azure Resource Group(s): Primary Resource Group and AKS Resource Group
-  >- Virtual Network, Network Security Groups and Network Security Rules
+  >- Azure resource group(s): primary resource group and AKS resource group
+  >- Virtual network, network security groups, and network security rules
   >- Managed Azure Kubernetes Service (AKS) cluster
-  >- System and User AKS Node pools with required Labels and Taints
-  >- Infrastructure to deploy SAS Viya CAS in SMP or MPP mode
-  >- Storage options for SAS Viya -  AzureDisk/Files(dev) or NFS Server or Azure NetApp Files(HA)
-  >- Azure DB for PostgreSQL, optional
-  >- Azure Container Registry, optional
+  >- System and user AKS node pools with required labels and taints
+  >- Infrastructure to deploy the SAS Viya CAS server in SMP or MPP mode
+  >- Storage options for SAS Viya:  Azure Premium Managed Disks (standard deployments); NFS Server or Azure NetApp Files (HA deployments)
+  >- Azure Database for PostgreSQL (optional)
+  >- Azure Container Registry (optional)
 
 [<img src="./docs/images/viya4-iac-azure-diag.png" alt="Architecture Diagram" width="750"/>](./docs/images/viya4-iac-azure-diag.png?raw=true)
 
-Once the cloud resources are provisioned, see the [viya4-deployment](https://github.com/sassoftware/viya4-deployment) repo to deploy SAS Viya 4 products. If you need more information on the SAS Viya 4 products refer to the official [SAS&reg; Viya&reg; 4 Operations](https://go.documentation.sas.com/?cdcId=itopscdc&cdcVersion=v_001LTS&docsetId=itopswlcm&docsetTarget=home.htm&locale=en) documentation for more details.
+This project helps you to automate the cluster-provisioning phase of SAS Viya deployment. To learn about all phases and options of the
```

Comment:
We've been told to use SAS Viya 4 in our docs for these projects and now the '4' is being removed. Have we, SAS, changed directions on this? If we have a version '5' and the code base is still relevant we can adjust. Just asking as this was a key sticking point when the repos were created. Hence their names `viay4-iac-xxxxxxx`